typescript/libs/lsp_client.py

```diff
@@ -26,7 +27,10 @@ class LspCommClient(node_client.CommClient):
     def __init__(self, binary_path, args, env, root_path):
         self.server_proc = None
         self.args = copy.deepcopy(args)
-        self.args.insert(0, binary_path)
+        program = LspCommClient.which(binary_path)
```

Comment:
What does this get us? Why bother calling which, as opposed to having the user hardcode in the address where the binary is (which they'll probably have to do anyways since Sublime Text 3 doesn't capture paths set up in .bashrc?

The which command below is quite a lot of code!